src/main/java/org/opensearch/flowframework/model/State.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.model;
+
+/**
+ * Enum relating to the state of a workflow
+ */
+public enum State {
+    NOT_STARTED,
+    PROVISIONING,
+    FAILED,
+    READY
```

Comment:
```suggestion
    COMPLETED
```

Reply:
I am not 100% sure what the correct state name would be here, but I originally went with `READY` since this is for the entire workflow, so while provisioning is complete the workflow itself is ready to be used for any ingestion or query requests later on by the orchestrate API. Could `ACTIVE` be better here?

Reply:
+1 to `Completed`

Reply:
I'll vote for COMPLETED.